packages/server-core-integration/package.json

```diff
@@ -74,6 +74,7 @@
 		"faye-websocket": "^0.11.4",
 		"got": "^11.8.5",
 		"tslib": "^2.3.1",
+		"winston": "^3.5.1",
```

Comment:
`winston` should be removed from the package.json of mos-gateway and playout-gateway with it defined here, or we will end up pulling in multiple versions